util/reggen/gen_dv.py

```diff
@@ -19,6 +20,28 @@
 from .window import Window
 
 
+class DvBaseNames:
+    # Class global attributes
+    valid_types = ["pkg", "block", "reg", "field", "mem", "all"]
+
+    def __init__(self, ip: str):
+        self.ip = ip
```

Comment:
Since the DvBaseNames is maintained as a Dict, maybe we do not need this attributes?

Reply:
Agreed.